consensus/src/epoch_manager.rs

```diff
@@ -308,14 +308,17 @@ impl EpochManager {
             Ok(()) => { /* Expected */ }
             Err(Error::ValidatorKeyNotFound(error)) => {
                 error!(
+                    epoch = epoch,
                     error = error,
-                    "Unable to find the validator consensus key during initialize",
+                    "Unable to find the validator consensus key during initialize() and will not participate in consensus.\
+                     Will continue syncing until the validator consensus key matches in a future epoch."
                 );
             }
             Err(Error::ValidatorNotInSet(error)) => {
                 error!(
+                    epoch = epoch,
                     error = error,
-                    "The validator is not in the current validator set",
+                    "The validator is not in the current validator set for the epoch and will not participate in consensus.",
```

Comment:
I don't think we need these additional errors as it is already defined in the internal error message.